homeassistant/components/unifi/controller.py

```diff
@@ -88,7 +88,6 @@ class UniFiController:
     def __init__(self, hass, config_entry):
         """Initialize the system."""
         self.hass = hass
-        self.config_entry = config_entry
```

Comment:
This shouldn't need to be moved?

Reply:
I think we do actually update the object instead of making a copy.  Will double check and remove if so

Reply:
No its the same, would have broken a long time ago else :)

Reply:
Fixed!